opus/application/apps/cart/templates/cart/cart.html

```diff
@@ -13,7 +13,11 @@
                     <span id="op-total-download-count">{{ total_download_count|intcomma }}</span>
                 </p>
             </div>
-            <div class="op-download-options">
+            <div class="op-download-options
+                    {% if count < 1 and recycled_count < 1 %}
```

Comment:
Shouldn't this just be based on count, not recycled_count? Even if there are things in the recycle bin, we don't want to allow people to download, since there's nothing in the main cart.